### ISSUE TITLE
Boxes on tracks for items not showing

### ISSUE THREAD

**Issue by JamesW-webdev** [Sk1mble/fate-core-official#60]
I've added an ad-hoc tracker, "# of Uses" as an example, to an item. It has 4 boxes and is set to escalating. The name of the track does pop up in the Tracks category of the item, however, the boxes do not show up whatsoever. I've tried messing with the tracker's settings, I've tried clicking in those areas where I think the boxes would be in case it's just a display issue, and I've had no luck.

**Comment by Sk1mble** [comment]
Tracks for extras are added to the character's tracks, not displayed
separately for the extra.

**Comment by JamesW-webdev** [comment]
Oh! I had no idea!

So just add the item to a character sheet and it'll show up then?

**Comment by Sk1mble** [comment]
Yes, as long as the extra is active. The same is true for its stunts,
aspects and skills.
